proto/storage/file_access.proto

```diff
@@ -8,14 +8,14 @@ import "storage/process_indicator.proto";
 option go_package = "./storage;storage";
 option java_package = "io.stackrox.proto.storage";
 
-// FileActivity contains fields related to arbitrary file activity
+// FileAccess contains fields related to arbitrary file accesses
```

Comment:
Just a clarification question(s):  Did this exist in 4.9 as `FileActivity`?  Did anything get persisted to the database using it as `FileActivity`?  

I don't think there was so the rename would be fine.  But I want to verify that with you just to make sure we don't miss a potential issue.